components/dashboard/src/Menu.tsx

```diff
@@ -227,8 +221,8 @@ export default function Menu() {
               ]
             : []),
         {
-            title: "Docs",
-            link: "https://www.gitpod.io/docs/",
+            title: "Workspaces",
+            link: "/workspaces",
         },
```

Comment:
Following up from https://github.com/gitpod-io/gitpod/pull/10309/files#r884587258:

```suggestion
            title: "Workspaces",
            link: "/workspaces",
            alternatives: ["/"],
        },
```

Reply:
Awesome! Works for me!